api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -75,13 +75,18 @@ public long getQueryBatchSize() {
   }
 
   @Override
-  public List<ReportingWorkspaceFreeTierUsage> getWorkspaceFreeTierUsage() {
+  public List<ReportingWorkspaceFreeTierUsage> getWorkspaceFreeTierUsageBatch(
```

Comment:
all batchifications here match the previously existing ones: add limit and offset